rust/worker/src/index/fulltext/types.rs

```diff
@@ -230,33 +282,47 @@ mod test {
 
     #[test]
     fn test_index_and_search_multiple_documents() {
-        let pl_blockfile = Box::new(HashMapBlockfile::open(&"in-memory-pl").unwrap());
-        let freq_blockfile = Box::new(HashMapBlockfile::open(&"in-memory-freqs").unwrap());
-        let tokenizer = Box::new(TantivyChromaTokenizer::new(Box::new(NgramTokenizer::new(1, 1, false).unwrap())));
-        let mut index = BlockfileFullTextIndex::new(pl_blockfile, freq_blockfile, tokenizer);
-        index.begin_transaction().unwrap();
-        index.add_document("hello world", 1).unwrap();
-        index.add_document("hello chroma", 2).unwrap();
-        index.add_document("chroma world", 3).unwrap();
-        index.commit_transaction().unwrap();
+        let mut provider = HashMapBlockfileProvider::new();
+        let pl_blockfile = provider
+            .create("pl", KeyType::String, ValueType::PositionalPostingList)
+            .unwrap();
+        // let freq_blockfile = provider
+        //     .create("freq", KeyType::String, ValueType::Int32)
```

Comment:
Is there a reason we commented this out? We can delete if it's not adding functionality

Reply:
i think this was an accident, my bad!